src/components/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Button = ({ label, onClick, isDisabled = false, className }) => {
+  return (
+    <button className={className} onClick={onClick} disabled={isDisabled}>
+      {label}
+    </button>
```

Comment:
Why the button component is not styled? Now it shows the basic state right?

Reply:
The button is used in the Navbar section and has been styled using its class name. Is a button style is needed ?